packages/react/src/components/List/List.tsx

```diff
@@ -0,0 +1,77 @@
+import { useMergeRefs } from '@floating-ui/react';
+import { Slot } from '@radix-ui/react-slot';
+import cl from 'clsx/lite';
+import { forwardRef, useEffect, useId, useRef } from 'react';
+
+import { Paragraph } from '../Typography';
+
+export type ListProps = {
+  /**
+   * Change the default rendered element for the one passed as a child, merging their props and behavior.
+   * @default false
+   */
+  asChild?: boolean;
+  /**
+   * Changes text sizing
+   * @default md
+   *
+   */
+  size?: 'sm' | 'md' | 'lg';
+  /**
+   * Changes type of list style
+   * @default unordered
+   *
+   */
+  variant?: 'unordered' | 'ordered';
```

Comment:
I feel this goes against making it more HTML-aligned.
Having `List.Ordered` and `List.Unordered` feels more like using the HTML element, than doing `variant="ordered"`.
The API would also look more like what your write in HTML:
```jsx
<List.Ordered> // ol
  <List.Item>List Item 3.1</List.Item> // li
  <List.Item>List Item 3.2</List.Item>
  <List.Item>List Item 3.3</List.Item>
</List.Ordered>
```

Reply:
Ja, enig - jeg ønsket egentlig å eksponere `Ul` og `Ol` men ble heller `List` etter prat med Mike men her kan dere bestemme :)

Reply:
Trur denne kan fint bruke `List.Ord/Uno`, sia det er eit faktisk HTML element som blir skifta mellom 😄